src/ogma/features/nodeGrouping.ts

```diff
@@ -163,7 +163,9 @@ export class NodeGroupingTransformation {
    * @param node reference to the virtual node
    */
   private _getNodeGroupingCaption(node: Node<LkNodeData> | undefined): string | undefined {
-    if (node !== undefined && node.isVirtual()) {
+    // TODO: Normally there is no need to check if getSubNodes return a value, Ogma issue
```

Comment:
Do we need to track the TODO somewhere once Ogma fixes the issue ?

Reply:
We don't have a way to do so. Normally when ogma team fix the issue they should put a comment in the ticket.
That way we will be notified.
If not we will check the TODO is the future and check the link to verify if it was fixed or not